### ISSUE TITLE
Logitech G305 not working

### ISSUE THREAD

**Issue by Interknet** [libratbag/libratbag#820]
ratbagctl list shows no devices.
I chaned the usb code to the one shown in lsusb.
Tried on kernel 4.19 and 5.3, no change.

Result of: **sudo ratbagd --verbose=raw**
ratbag debug: New device: Microsoft Microsoft® SiderWinderTM X4 Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 045e:0768
ratbag debug: New device: Microsoft Microsoft® SiderWinderTM X4 Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 045e:0768
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 08
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: - HID report ID 20
ratbag debug: - HID report ID 21
ratbag raw: hidpp write:  11 ff 00 18 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag raw: hidpp read:   10 ff 8f 00 18 01 00
ratbag debug:     HID++ error from the device (255): ERR_UNKNOWN (01)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 08
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: - HID report ID 20
ratbag debug: - HID report ID 21
ratbag raw: hidpp write:  11 ff 00 18 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag raw: hidpp read:   10 ff 8f 00 18 01 00
ratbag debug:     HID++ error from the device (255): ERR_UNKNOWN (01)
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver hidpp20
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw3'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 08
ratbag debug: hidraw info: bus 0x03 vendor 0x46d product 0xffffc53f
ratbag debug: Logitech USB Receiver is device '/dev/hidraw4'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 10
ratbag debug: - HID report ID 11
ratbag debug: - HID report ID 20
ratbag debug: - HID report ID 21
ratbag raw: hidpp write:  11 ff 00 18 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag raw: hidpp read:   10 ff 8f 00 18 01 00
ratbag debug:     HID++ error from the device (255): ERR_UNKNOWN (01)

**Comment by persson1589** [comment]
I have the same issue, can confirm that the g305 is not working
ratbagd version 0.10-1-201909111032~ubuntu19.04.1

sudo ratbagd --verbose=raw
ratbag debug: New device: Kingston HyperX Alloy FPS Mechanical Gaming Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0951:16b7
ratbag debug: New device: Kingston HyperX Alloy FPS Mechanical Gaming Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0951:16b7
ratbag debug: New device: Kingston HyperX Alloy FPS Mechanical Gaming Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0951:16b7
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c53f
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c53f
ratbag debug: New device: Logitech USB Receiver
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 046d:c53f
ratbag debug: New device: Corsair Components Inc. Corsair HS60 Surround Adapter
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1b1c:0a3b
ratbag debug: New device: BLUE MICROPHONE Blue Snowball 
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 0d8c:0005

**Comment by FFY00** [comment]
Closed in favor of #595

TLDR: You need Linux 5.4 once it's out.
